server/config/environment/shared.js

```diff
@@ -5,31 +5,37 @@ var configExport = {
   ENERGY_TYPES : {
     COAL : {
       name: 'Coal',
+      key: 'coal',
```

Comment:
Could we use the actual key .toLowerCase() instead? (Happy to change from SNAKE_CASE to lower too.